src/main/java/com/breskul/bibernate/util/EntityUtil.java

```diff
@@ -47,6 +48,14 @@ public static Field findEntityIdField(List<Field> fields) {
     return idFields.get(0);
   }
 
+  public static String findEntityIdFieldName(Class<?> entityClass) {
```

Comment:
I think need merge with method above, just overload method and call method wiche get list of field as parameter

Reply:
Or remove one

Reply:
done